eks/iam.tf

```diff
@@ -262,3 +262,44 @@ resource "aws_iam_policy" "efs_policy" {
 }
 EOF
 }
+
+
+module "iam_assumable_role_cluster_autoscaler" {
+  source           = "terraform-aws-modules/iam/aws//modules/iam-assumable-role-with-oidc"
+  version          = "3.6.0"
+  create_role      = true
+  role_name        = "efs-driver"
+  provider_url     = replace(aws_iam_openid_connect_provider.cluster.url, "https://", "")
+  role_policy_arns = [aws_iam_policy.cluster_autoscaler_policy.arn]
+  # namespace and service account name
+  oidc_fully_qualified_subjects = ["system:serviceaccount:kube-system:cluster-autoscaler"]
```

Comment:
Is there a way to reference this?

Reply:
we can make a variable like `ca-serice-account-name` and use it here and helm chart

Reply:
added the variable for service account